mcserver/utils.py

```diff
@@ -1,5 +1,34 @@
 from django.conf import settings
 from django.template.loader import render_to_string
+import boto3
+
+
+def submit_custom_metric(namespace, metric_name, value):
+    """
+    Submit a custom metric to AWS CloudWatch.
+
+    Parameters:
+    - namespace (str): The namespace for the metric data.
+    - metric_name (str): The name of the metric.
+    - value (float): The value associated with the metric.
+    """
+    client = boto3.client(
+        'cloudwatch',
+        aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
```

Comment:
You don't need to explicitly specify these because I [assigned task role to the celery task](https://github.com/stanfordnmbl/opencap-infrastructure/pull/15/files#diff-1213bd78925548ce5f8721be601df0d9b67522d0a0d3ba09f4f0a235f3877cc1R142) which will allow boto3 to infer the credentials from environment.